lib/common.py

```diff
@@ -505,7 +511,7 @@ def all_vdis_on_host(self, host):
                 return False
         return True
 
-    def export(self, filepath, compress="false"):
+    def export(self, filepath, compress=False):
```

Comment:
This is a special case. `compress` may be 'false', 'true' or 'zstd', so maybe we shouldn't treat it as a boolean here and keep using a string instead.

Reply:
IHMO this parameter name `compress` is a source of error when you don't know the API, it really looks like a boolean. A function parameter like `compressMode` is already more precise using values like: "None", "Default" (or something else), "Zstd"...

Reply:
Ok, so we can use directly: `gzip`, `zstd` or anything else like `none`: https://github.com/xapi-project/xen-api/blob/550ff27340167239b33aa41b2f517c7771df9213/ocaml/xapi/export.ml#L821-L826

Reply:
Perfect